diff -x *.pyc -x *~ -r -u ../../../src/Pale-Moon/platform/build/moz.configure/old.configure platform/build/moz.configure/old.configure
--- ../../../src/Pale-Moon/platform/build/moz.configure/old.configure	2023-11-17 10:29:06
+++ platform/build/moz.configure/old.configure	2024-01-27 16:03:58
@@ -316,6 +316,11 @@
     '--disable-browser-statusbar',
     '--disable-sync',
     '--disable-personas',
+    '--enable-cuemol',
+    '--enable-cuemol-deplibs',
+    '--enable-cuemol-debug',
+    '--enable-cuemol-boostdir',
+    '--enable-cuemol-bundle',
     
     # Below are configure flags used by Basilisk
     '--disable-webextensions',
