--- platform/build/moz.configure/old.configure	2023-11-04 15:47:00
+++ ../../../src/uxp_example1/platform/build/moz.configure/old.configure	2023-12-27 21:05:54
@@ -316,6 +316,8 @@
     '--disable-browser-statusbar',
     '--disable-sync',
     '--disable-personas',
+    '--enable-cuemol',
+    '--enable-cuemol-deplibs',
     
     # Below are configure flags used by Basilisk
     '--disable-webextensions',
